### ISSUE TITLE
Normal vs Extended

### ISSUE THREAD

**Issue by drego85** [lightswitch05/hosts#64]
Hi Daniel,
can you kindly tell me the differences between the normal lists (eg ads-and-tracking.txt) and the extended ones (eg ads-and-tracking-extended.txt)?

Thank you
Andrea

**Comment by lightswitch05** [comment]
Hey sure Andrea @drego85 

`ads-and-tracking.txt` are hosts that I've found to be serving ads or tracking, depending on the domain, I might add both a subdomain and the root domain. For example, if I see `ad-server-3.ad-and-tracking-company.example`, I'll normally make two entries in `ads-and-tracking.txt`:

```
0.0.0.0 ad-and-tracking-company.example
0.0.0.0 ad-server-3.ad-and-tracking-company.example
```

However, you can guess by looking at the subdomain `ad-server-3` that they most likely have more then one ad server. That is where the extended list comes into effect. When I add a domain to the normal base list, I have code that attempts to automatically discover more subdomains for any blocked domains. In this example, it wouldn't find anything else for `ad-server-3.ad-and-tracking-company.example`, but since I also added `ad-and-tracking-company.example`, there is a good chance that it might be able to discover the other subdomains, making the extended list look like this:

```
0.0.0.0 ad-and-tracking-company.example
0.0.0.0 ad-server-1.ad-and-tracking-company.example
0.0.0.0 ad-server-2.ad-and-tracking-company.example
0.0.0.0 ad-server-3.ad-and-tracking-company.example
0.0.0.0 ad-server-4.ad-and-tracking-company.example
```

This method of expanding domains does have some drawback with list size and false positives. However, since it only works on finding subdomains - not sibling domains - its pretty reliable. I try to be careful to only add a root domain when it is clear that everything coming from a domain should be blocked. Anyways, I personally use the extended version, and that is the one I recommend people to use. But if you want a smaller list, the normal base list is available. I hope that answers your questions!

-Daniel

**Comment by drego85** [comment]
Wow, fast and accurate! 

Thank you so much!

I advise you to specify this difference in the initial README, probably I'm not the only one who has this doubt!

**Comment by lightswitch05** [comment]
Yes, I used to have all the listed in the README is a description. I eventually decided to remove the base lists and to just recommended the extended versions- but you are right that a description would still be useful. If I were starting out this project again, I would hide the base lists into a sources folder and ditch the 'extended' naming structure. Since people are already using the lists, I don't want to break links just yet. I have this issue pinned now. Lets leave it open to make it easy for others to find
